docs/ops/condition/NonZero_3.md

```diff
@@ -4,34 +4,37 @@
 
 **Category**: Condition operation
 
-**Short description**: *NonZero* returns the indices of the non-zero elements of the input tensor.
+**Short description**: *NonZero* returns the indices of the non-zero elements of the input tensor. 
 
 **Detailed description**: *NonZero* returns the indices of the non-zero elements of the input tensor (in row-major order - by dimension).
-The output tensor has shape `[rank(input), num_non_zero]`. For example, for the tensor `[[1, 0], [1, 1]]` the output will be `[[0, 1, 1], [0, 0, 1]]`.
+* The output tensor has shape `[rank(input), num_non_zero]`.
+* For example, for the tensor `[[1, 0], [1, 1]]` the output will be `[[0, 1, 1], [0, 0, 1]]`.
+* Each output column represents a single non-zero element and for that column, value in row `i` represents this element's index in input's `i`'th dimension.
+* In given example the columns would be: `[0,0]`,`[1,0]`,`[1,1]`.
 
 **Attributes**
-
 * *output_type*
-
   * **Description**: the output tensor type
-  * **Range of values**: "i64" or "i32"
+  * **Range of values**:
+    * `i64`
+    * `i32`
```

Comment:
Please, change to 
```
* **Range of values**: `i64` or `i32`
```
I don't see any reason why we should spend 2 lines instead of one